src/components/AccordionItem/AccordionItem.module.css

```diff
@@ -0,0 +1,19 @@
+/*------------------------------------*\
+    # ACCORDION ITEM
```

Comment:
Name suggestions? Accordion Item v. Accordion Row v. Accordion Section?

Reply:
I would prefer "section" or "row" over "item". For some reason "section" sounds right to me, but I'm also fine with "row".

Reply:
Will go with row since that term is also used in the mocks